File: src/main/java/me/miki/shindo/management/event/impl/EventZoomFov.java
package me.miki.shindo.management.event.impl;

import me.miki.shindo.management.event.Event;

public class EventZoomFov extends Event {

    private float fov;

    public EventZoomFov(float fov) {
        this.fov = fov;
    }

    public float getFov() {
        return fov;
    }

    public void setFov(float fov) {
        this.fov = fov;
    }
}

File: src/main/kotlin/me/miki/shindo/management/event/impl/EventZoomFov.kt
package me.miki.shindo.management.event.impl

import me.miki.shindo.management.event.Event

class EventZoomFov(
    private var fov: Float,
) : Event() {
    fun getFov(): Float = fov

    fun setFov(fov: Float) {
        this.fov = fov
    }
}
